docs/development/design_guide/typography.md

```diff
@@ -9,9 +9,7 @@ In case the text refers to an inactive element, the grey text color should be us
 The font should always be the [Quicksand font](https://fonts.google.com/specimen/Quicksand), as the customers prefer that the font is similar to how the citizens learn to write, especially in relation to the letter a.
 
 ## Font size
-The font size should differ, depending on whether the application is in portrait or landscape mode.
-In portrait mode, the text should be 30 pt, and in landscape it should be 20 pt.
-Additionally, the package [auto_size_text](https://pub.dartlang.org/packages/auto_size_text) should be used to ensure that the text fits within its area.
+The font size should follow the sizes in file FontSizes(Placeholder).
```

Comment:
Why do we want to state this before creating the file? (especially since the problem does not address this)
Couldn't we create a separate issue for this and handle it (i.e. creating this "FontSizes" file) before making the changes in the wiki page?
What I mean is: is it not better to at least have some sort of design guideline for the Font size until this file is created (i.e. better to use the old design guidelines until the new ones are made, than delete them and state that there will be guidelines in a file at some point)?